astropy/units/quantity.py

```diff
@@ -1722,3 +1722,50 @@ def _set_unit(self, unit):
                  ", so cannot set it to '{0}'.".format(unit)))
 
         super()._set_unit(unit)
+
+
+def allclose(a, b, rtol=1.e-5, atol=None, **kwargs):
+    """
+    Notes
+    -----
+    Returns True if two arrays are element-wise equal within a tolerance.
+
+    This is a :class:`~astropy.units.Quantity`-aware version of
+    :func:`numpy.allclose`.
+    """
+    return np.allclose(*_unquantify_allclose_arguments(a, b, rtol, atol),
+                       **kwargs)
+
+
+def _unquantify_allclose_arguments(actual, desired, rtol, atol):
+    from .. import units as u
```

Comment:
We are inside `units` here, so no need for this import; just change `u.Quantity` -> `Quantity` below (and same for `UnitsError`)